DDI_Qualification_CYP3A4/report.md

```diff
@@ -88,7 +102,7 @@ This **qualification report** evaluates for the PBPK platform **PK-Sim** (as par
 
 To demonstrate the level of confidence, the predictive performance of the platform for this indented purpose is assessed via a network of PBPK models of selected index CYP3A4 DDI perpetrators (covering the range from strong induction to strong inhibition), and respective sensitive index CYP3A4 victim drugs and a comprehensive dataset from published clinical DDI studies. All PBPK models represent whole-body PBPK models, which allow dynamic DDI simulations in organs expressing CYP3A4. 
 
-The respective *qualification plan* to produce this *qualification report* is transparently documented and provided open-source (https://github.com/Open-Systems-Pharmacology/OSP-Qualification-Reports). The same applies for all presented PBPK models including *evaluation reports* on model building and evaluation of each model (https://github.com/Open-Systems-Pharmacology/OSP-PBPK-Model-Library).
+The respective *qualification plan* to produce this *qualification report* is transparently provided open-source (https://github.com/Open-Systems-Pharmacology/OSP-Qualification-Reports). The same applies for all presented PBPK models including *evaluation reports* on model building and evaluation of each model (https://github.com/Open-Systems-Pharmacology/OSP-PBPK-Model-Library).
```

Comment:
@sfrechen This is not quite correct, right? Qualification **plan** is not in the OSP-Qualification-Reports repo but in the "Qualification-CYP3A4-DDI"-repo and models evaluation **plans** are not in the OSP-PBPK-Model-Library repo but in the models repos.

Reply:
Thx @Yuri05 
fixed with updated commits

Reply:
qualification plan repository was fixed with https://github.com/Open-Systems-Pharmacology/Qualification-DDI-CYP3A4/commit/f4214a324b9ed5333c4dcfbd59b3b10310362cc5